NitroxServer/Communication/Packets/Processors/AggressiveWhenSeeTargetChangedProcessor.cs

```diff
@@ -0,0 +1,13 @@
+using NitroxModel.Packets;
+using NitroxServer.Communication.Packets.Processors.Abstract;
+using NitroxServer.GameLogic;
+using NitroxServer.GameLogic.Entities;
+
+namespace NitroxServer.Communication.Packets.Processors;
```

Comment:
Maybe we could regroup the different processors related to creature sync under a same folder ?
Thoughts on it ?